Dispatcher/UI/Screens/HomePage/HomePageViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class HomePageViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+    var viewModel: HomePageViewModel = HomePageViewModel(repository: MockArticleRepository())
+    var mockArticles:[Card] = []
+    let navigationBar = NavigationBar()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        mockArticles = viewModel.repository.getArticles()
```

Comment:
this is violation the law of demeter programming principle
(we will talk about it next lesson)
you can read about it here meanwhile
https://daddycoding.com/2023/03/27/the-law-of-demeter/